### PR TITLE
Data validation for raw data

### LINKED ISSUE #127: Validate raw data
Check for consistency in dtypes in columns etc (some of the automated tools mentioned in https://github.com/Aarhus-Psychiatry-Research/psycop-t2d/issues/150 do the trick) if not already done? 

- [ ] Check all loaders once with e.g. deepchecks before committing? 

Somewhat aligns with #78

### PR DESCRIPTION
**Pull request by HLasse**
- [X] I have battle-tested on Overtaci (RMAPPS1279)
- [X] I have bumped the version as described in the wiki

## Notes for reviewers
Adds data validation for raw data sets (or any df). Focus on `data_checks/raw_data_validation`. Ideas for better heuristics for coloring outlying rows are welcome

Fixes #127.

### CODE REVIEW

File: src/psycopmlutils/feature_describer/feature_describer.py [hunk 1]
```diff
@@ -103,24 +111,25 @@ def generate_feature_description_row(
...
+        "N unique": series.nunique(),
+        "Fallback strategy": predictor_dict["fallback"],
+        "Proportion missing": series.isna().mean(),
+        "Mean": round(series.mean(), 2),
+        "Histogram": create_unicode_hist(series),
+        "Proportion using fallback": get_value_proportion(
+            series,
+            predictor_dict["fallback"],
+        ),
+    }
```
Comment (MartinBernstorff) [inline src/psycopmlutils/feature_describer/feature_describer.py:127]: This function has very large overlap with `generate_column_description` now. Can we refactor?

**Review by MartinBernstorff — APPROVED** [review]
Looks great Lasse! A few comments, then I think it's ready for merge.

File: src/psycopmlutils/data_checks/validate_raw_data.py [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+    timestamp_col_name = "timestamp" if "timestamp" in df.columns else None
+    id_col_name = "dw_ek_borger" if "dw_ek_borger" in df.columns else None
+
+    # Deepchecks
+    ds = Dataset(df=df, index_name=id_col_name, datetime_name=timestamp_col_name)
+    integ_suite = data_integrity(timeout=0)
+    with msg.loading("Running data integrity checks..."):
+        suite_results = integ_suite.run(ds)
+        suite_results.save_as_html(str(savepath / "data_integrity.html"))
+        failed_checks["data_integrity"] = get_name_of_failed_checks(suite_results)
```
[the remark this answers is not in the document]
Comment (HLasse) [reply src/psycopmlutils/data_checks/validate_raw_data.py:59]: No, as they might be useful for the raw data (less standardized)

File: src/psycopmlutils/feature_describer/feature_describer.py (hunk 1 above)
Comment (HLasse) [reply src/psycopmlutils/feature_describer/feature_describer.py:127]: This one requires predictor df, so we won't merge the two (as dicussed yesterday)

**Pull request merged**